lib/Echidna/Campaign.hs

```diff
@@ -157,13 +166,18 @@ execTxOptC t = do
 -- | Given an initial 'VM' and 'World' state and a number of calls to generate, generate that many calls,
 -- constantly checking if we've solved any tests or can shrink known solves. Update coverage as a result
```

Comment:
Can you update the comments here and/or add comments in the new lines of `callseq`. This function is starting to be very difficult to understand..